Client/src/Components/DatasetUpload/DatasetView.tsx

```diff
@@ -68,7 +71,12 @@ export const DatasetView = (props: IProps) => {
   }, [])
 
   const handleSubmitForm = async (formDataset: IDatasetModel) => {
-    await callSaveDataset(formDataset)
+    const result = await callSaveDataset(formDataset)
+    if (result > 0) {
+      SnackbarUtils.success("Dataset successfully uploaded")
+      history.push('/dataset/' + result)
+      //history.go(0)
```

Comment:
Same with this line

Reply:
it was removed in the latest push @Mehrdadmaskull